wagtail/contrib/modeladmin/templatetags/modeladmin_tags.py

```diff
@@ -156,8 +156,18 @@ def admin_list_filter(view, spec):
 def result_row_display(context, index):
     obj = context['object_list'][index]
     view = context['view']
+    row_attrs_dict = view.model_admin.get_extra_attrs_for_row(obj, context)
+    row_attrs_dict['data-object-pk'] = obj.pk
+    odd_or_even = 'odd' if (index % 2 == 0) else 'even'
+    if 'class' in row_attrs_dict:
+        row_attrs_dict['class'] += ' %s' % odd_or_even
+    else:
+        row_attrs_dict['class'] = odd_or_even
+    row_attrs = ''.join(
+        ' %s="%s"' % (key, val) for key, val in row_attrs_dict.items())
```

Comment:
(We already use `flatatt` in at least seven other places in Wagtail, one more wouldn't hurt)